en/04_Changelogs/5.1.0.md

```diff
@@ -48,6 +48,7 @@ Read more about [eager loading](/developer_guides/model/relations/#eager-loading
 
 - [`DataList::filterAny()`](api:SilverStripe\ORM\DataList::filterAny()) queries on many-many relations that use an aggregate `HAVING` clause now correctly use an `OR` conjunction rather than an incorrect `AND` conjunction.
 - At some point shortly before the release of Silverstripe CMS 4.0.0, SSL support for database connections was accidentally removed. This has now been reinstated - see [Using SSL in database connections](/developer_guides/security/secure_coding#using-ssl-in-database-connections) for more information.
+- The `cascade_duplicates` property was added to the [`InheritedPermissionsExtension`](api:SilverStripe\Security\InheritedPermissionsExtension) class and now when duplicating any object that applied `InheritedPermissionsExtension` the values in the `ViewerGroups` and `EditGroups` mapping tables will be copied so that the new copies of original records are related to the new duplicate record.
```

Comment:
```suggestion
- The `cascade_duplicates` property was added to the [`InheritedPermissionsExtension`](api:SilverStripe\Security\InheritedPermissionsExtension) class so that now when duplicating any object that has the `InheritedPermissionsExtension` applied the values in the `ViewerGroups` and `EditGroups` mapping tables will also be duplicated so that new object retains the same viewer and editor groups as the original.
```

Reply:
Updated